app/models/generic_file.rb

```diff
@@ -1,15 +1,15 @@
 class GenericFile < ActiveFedora::Base
   include Sufia::GenericFile
 
-  property :production_id, predicate: ::RDF::URI('http://docs.osfashland.org/terms/production_id'), multiple: false
+  property :production_ids, predicate: ::RDF::URI('http://docs.osfashland.org/terms/production_ids'), multiple: true
 
-  property :production_name, predicate: ::RDF::URI('http://docs.osfashland.org/terms/production_name'), multiple: false do |index|
+  property :production_names, predicate: ::RDF::URI('http://docs.osfashland.org/terms/production_names'), multiple: true do |index|
     index.as :stored_searchable, :facetable
   end
 
-  property :venue_id, predicate: ::RDF::URI('http://docs.osfashland.org/terms/venue_id'), multiple: false
+  property :venue_ids, predicate: ::RDF::URI('http://docs.osfashland.org/terms/venue_ids'), multiple: true
```

Comment:
Line is too long. [106/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.